measurements/api/measurements.py

```diff
@@ -106,13 +86,6 @@ def api_list_report_files():
     if since_index:
         q = q.filter(Report.report_no > report_no)
 
-    # XXX these are duplicated above, refactor into function
-    if order.lower() not in ('asc', 'desc'):
-        raise BadRequest("Invalid order")
-    if order_by not in ('test_start_time', 'probe_cc', 'report_id',
-                        'test_name', 'probe_asn', 'report_no'):
-        raise BadRequest("Invalid order_by")
-
```

Comment:
These checks have vanished. I am going to assume they're now embedded in the `YAML` file.

- [ ] make sure that's really the case.